### DISCUSSION TITLE
Chord Memory

### DISCUSSION THREAD

**Discussion by Dust-r** [SynthstromAudible/DelugeFirmware#1856] [Ideas]
I really like the new Chords Memory function in 1.1; can all of the side buttons have this Chord Memory functionality? Currently, only one row seems to offer this.

**Comment by Dust-r** [comment]
I'd like to play Jobim's "O Grande Amor" which has more than a dozen different chords with the new Chord Memory function, for example.

**Comment by jfs630** [comment]
Regarding "SIDEBAR CONTROLS" added in c1.1.0 Beethoven, currently the same feature can only be set for one column, but I agree with the idea to allow this to be set for both columns.

For example, for VELOCITY, if it can only be set for up to one column,

127
111
95
79
63
47
31
15

↑As shown above, only 8 steps from 0 to 127 in increments of 16 can be displayed at a time.
However, if both of the two columns can be set to velocity,

127
119
111
103
95
87
79
71

63
55
47
39
31
23
15
7

↑As shown above, it is now possible to display from 0 to 127, up to 16 steps in increments of 8 at a time, which will be comfortable for users who wish to edit velocity in detail.
This is true not only for VELOCITY, but also for MOD WHEEL, CHORDS, CHORD MEMORY, and SCALES.

**Comment by Dust-r** [comment]
> Regarding "SIDEBAR CONTROLS" added in c1.1.0 Beethoven, currently the same feature can only be set for one column, but I agree with the idea to allow this to be set for both columns.
> 
> For example, for VELOCITY, if it can only be set for up to one column,
> 
> 127 111 95 79 63 47 31 15
> 
> ↑As shown above, only 8 steps from 0 to 127 in increments of 16 can be displayed at a time. However, if both of the two columns can be set to velocity,
> 
> 127 119 111 103 95 87 79 71
> 
> 63 55 47 39 31 23 15 7
> 
> ↑As shown above, it is now possible to display from 0 to 127, up to 16 steps in increments of 8 at a time, which will be comfortable for users who wish to edit velocity in detail. This is true not only for VELOCITY, but also for MOD WHEEL, CHORDS, CHORD MEMORY, and SCALES.

Cool idea! If Chord Memory was expanded I'd be happy.

**Reply by soymonitus** [reply]
In nightly you have 2 different chord memory banks, the Song one (shared across all clips, same as the one in 1.1) and the per-clip one (new in nightly), so basically you can have 16 different chords at the same time using those two columns

**Reply by Dust-r** [reply]
Thanks, soymonitus. I don't understand the difference between Song chord memory banks and Per-clip chord memory banks, but I will try to check it out the nightly later; maybe I can figure it out.

**Comment by Delugional** [comment]
Request for chords to be mappable to MIDI CC (not Notes). 
(Hope I put this request in the right place)

**Reply by nikodemus** [reply]
You probably want to make a separate one for this.
